internal/matrix/mautrix_client_test.go

```diff
@@ -0,0 +1,34 @@
+package matrix
+
+import (
+	. "neurobot/domain/message"
```

Comment:
I didn't knew about `.` in import but found out that its recommended to use `_` instead of `.` in such cases